backend/app/api/endpoints/adapter/retrievers.py

```
@@ -284,14 +287,15 @@ async def test_retriever_connection(
         gateway = get_query_gateway()
-        try:
-            return await gateway.test_connection(runtime_spec)
-        except RemoteRagGatewayError:
-            return await LocalRagGateway().test_connection(runtime_spec)
+        return await gateway.test_connection(runtime_spec)
 
     except ValueError as e:
         return {"success": False, "message": str(e)}
 
+    except RemoteRagGatewayError as e:
+        logger.error(f"Retriever connection test failed: {str(e)}")
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)